Fix wwgetnodeconfig error when booting

fun2yang opened the pull request:
The wwgetnodeconfig and other scripts starts with wwget* will write logs into '/var/log/warewulf/provision/http.log'.  However, the 'provision' directory is not exist, which causes the error.  So, create it before exec the script.

jmstover commented on provision/initramfs/init:
```diff
@@ -480,9 +480,8 @@ for i in /etc/warewulf/init/*; do
 done
 
-
+mkdir -p /var/log/warewulf/provision
```
Can you move this up to just above the:
```
msg_blue "Now Booting Warewulf...\n"
```
line? Just make the directory at the start of the init file.

fun2yang replied:
OK, changed.

jmstover replied:
Thanks.

fun2yang committed:
Fix wwgetnodeconfig error when booting

The wwgetnodeconfig and other scripts starts with wwget* will write logs
into '/var/log/warewulf/provision/http.log'. However, the 'provision'
directory is not exist, which causes the error. So, create it before
exec the script.

Signed-off-by: Yang Jian <yangj.fnst@cn.fujitsu.com>

jmstover commented:
LGTM.

**Edit:**
Doh! Didn't mean to close it... *whistles innocently*

jmstover committed:
Merge branch 'development' into fix-wwgetnodeconfig-error
